fixtures/lep-laser-entertainment-products/diamond-pro-2-8.json

```diff
@@ -0,0 +1,906 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "diamond Pro 2.8",
+  "categories": ["Laser"],
+  "meta": {
+    "authors": ["Dirk", "dirk"],
+    "createDate": "2019-10-04",
+    "lastModifyDate": "2019-10-04",
+    "importPlugin": {
+      "plugin": "qlcplus_4.12.1",
+      "date": "2019-10-04",
+      "comment": "created by Q Light Controller Plus (version 4.12.2)"
+    }
+  },
+  "physical": {
+    "dimensions": [334, 125, 390],
+    "weight": 2.8,
+    "power": 50,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "Other"
+    },
+    "lens": {
+      "name": "Other"
+    }
+  },
+  "availableChannels": {
+    "Mode Select": {
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 63],
+          "type": "Maintenance",
+          "comment": "Laser Off"
+        },
+        {
+          "dmxRange": [64, 127],
+          "type": "Maintenance",
+          "comment": "Sound mode, shutoff after 2 sec"
+        },
+        {
+          "dmxRange": [128, 159],
+          "type": "Maintenance",
+          "comment": "Sound mode, (beam patterns)"
+        },
+        {
+          "dmxRange": [160, 191],
+          "type": "Maintenance",
+          "comment": "Sound mode, (graphics)"
+        },
+        {
+          "dmxRange": [192, 255],
+          "type": "Maintenance",
+          "comment": "DMX Mode"
+        }
+      ]
+    },
+    "Pattern Selection": {
+      "defaultValue": 0,
+      "capabilities": [
```

Comment:
"capabilities": {

Reply:
This is fine. There are multiple capabilities, so this is an array (`[]`) instead of a single object (`{}`).